### ISSUE TITLE
EPIC - Profile Completion

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/web#3954]
### Scenarios

As Gitcoin we'd like to be able to capture and display the most accurate information about the users of our platform.

As a contributor, I'd like Gitcoin to understand the full picture of who I am as a developer.

As a funder, I'd like to have a place to understand who I would potentially be hiring for my bounty.

### Stories Summary

- Define elements that make up a profile. What do we get from Github, what's editable/not.
- Design no data for profile completion
- Design profile completion experience
- Design editing experience profile if user chooses to finish later
- Design error cases for profile completion
- Design email notification for profile completion 
- Build no data for profile completion
- Build profile completion experience
- Build editing experience profile if user chooses to finish later
- Build error cases for profile completion
- Build email for profile completion 
- Add notification for profile completion
- Define data tracking for profile completion.
- Add data tracking for profile completion

**Comment by owocki** [comment]
for profile completion, we should decide what we want to ask for. i think this is my list

connect with github
everything in onboarding - 40%
avatar - 40%
do your first bounty - 20%
what else??

we will also need a design for the profile completion module.  something like this: 
![LinkedIn-Profile-Strength](https://user-images.githubusercontent.com/513929/54217343-109aee80-44b1-11e9-8312-add872d50fa4.jpg)

**Comment by owocki** [comment]
I think that 'profile completion' is maybe the wrong way to view this epic.  Instead, the meter should be framed as measuring how far along each user is in one of the two journeys:

1. As a coder, I want funding.
2. As a funder, I want coders.

In that vein, all of our products solve one of these two use cases so Ibelieve that we could/should have an area where we cross-sell our products to each other audience.

Basically I think this look like a module where each of the products are listed

- Bounties
- Grants
- Kudos
- CodeFund

and there is context explaining to an end user (1) how much / if they've used each product to reach their goal, and (1) how each one can be further used to connect them to funders or coders.  

@PixelantDesign feedback on this route?
